Formula/cdrtools.rb

```diff
@@ -1,14 +1,15 @@
 class Cdrtools < Formula
   desc "CD/DVD/Blu-ray premastering and recording software"
   homepage "http://cdrecord.org/"
-  url "https://downloads.sourceforge.net/project/cdrtools/cdrtools-3.01.tar.bz2"
-  mirror "https://fossies.org/linux/misc/cdrtools-3.01.tar.bz2"
-  sha256 "ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f"
-  revision 1
+  url "https://downloads.sourceforge.net/project/cdrtools/alpha/cdrtools-3.02a09.tar.gz"
+  mirror "https://fossies.org/linux/misc/cdrtools-3.02a09.tar.gz"
+  sha256 "c7e4f732fb299e9b5d836629dadf5512aa5e6a5624ff438ceb1d056f4dcb07c2"
 
```

Comment:
```suggestion
  license "CDDL-1.0"

```
http://cdrtools.sourceforge.net/private/linux-dist.html

Reply:
I almost made that change myself, but if you look at the `COPYING` file in the package there is a mix of CDDL/GPL/LGPL/BSD among various subcomponents.  It's a mess I didn't feel confident about adjudicating